### PR TITLE
Add TinyMS docker build script

### PR DESCRIPTION
**Pull request by leonwanghui**

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## TinyMS Dockerfile Repository
+
+This folder hosts all the `Dockerfile` to build TinyMS container images with various versions and hardware platforms.
```
Comment (hannibalhuang) [inline docker/README.md:3]: 'Dockerfile's....and corresponding hardware backends

File: docker/tinyms/0.1.0/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+ARG BASE_CONTAINER=mindspore/mindspore-cpu:1.1.1
+FROM $BASE_CONTAINER
+
+LABEL MAINTAINER="Leon Wang <wanghui71leon@gmail.com>"
+
+# Install base tools
+RUN apt update
```
Comment (hannibalhuang) [inline docker/tinyms/0.1.0/Dockerfile:7]: apt get update ?

**Review by hannibalhuang — CHANGES_REQUESTED** [review]
small fixes needed

File: docker/tinyms/0.1.0/Dockerfile (hunk 2 above)
Comment (leonwanghui) [reply docker/tinyms/0.1.0/Dockerfile:7]: In Ubuntu 18.04 version, `apt` equals with `apt-get`.

File: docker/README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ## TinyMS Dockerfile Repository
 
-This folder hosts all the `Dockerfile` to build TinyMS container images with various versions and hardware platforms.
+This folder hosts all the `Dockerfile` to build TinyMS container images with various versions and corresponding hardware platforms.
```
Comment (hannibalhuang) [inline docker/README.md:3]: s/platform/backends

File: docker/README.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
 
-> **NOTICE:** Currently TinyMS only support `CPU` hardware platform.
+> **NOTICE:** Currently TinyMS only supports `CPU` hardware platform.
```
Comment (hannibalhuang) [inline docker/README.md:5]: s/platform/backends

**Review by hannibalhuang — CHANGES_REQUESTED** [review]
small nixes

**Review by hannibalhuang — APPROVED** [review]
LGTM

**Review by lyd911 — APPROVED** [review]
lgtm

**Pull request merged**